### PR TITLE
content centering

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/karmada-io/website/issues/44

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -128,4 +128,8 @@ article header h1 {
   .support-wrapper li {
     padding: 10px 15px;
   }
+}
+
+.text-center {
+  text-align: center;
 }
```
Comment (RainbowMango) [inline src/css/custom.css:135]: No new line end of the file.

Comment (rgrupesh) [inline src/css/custom.css:135]: @Arhell Please add extra line at the end of the file.

Comment (Arhell) [reply src/css/custom.css]: Done

Comment (Arhell) [reply src/css/custom.css]: Done

**Comment by RainbowMango** [comment]
/approve

Leave LGTM to @rgrupesh for the final decision.

**Pull request merged**